clients/web/src/components/sources/source-pane.tsx

```diff
@@ -3,7 +3,7 @@ import { ImageView } from "~/components/sources/image-view.tsx";
 import { Match, Show, Suspense, Switch } from "solid-js";
 import { useParams, useSearchParams } from "@solidjs/router";
 import { decodeFileName, guessContentType } from "~/lib/sources/file-entries";
-import { CodeView } from "~/components/sources/code-view";
+import CodeView from "./code-view";
```

Comment:
curious about this change.
named exports usually allow for easier tree-shaking and code refactoring.

Why did you decide this component to use a default?